cmd/sonobuoy/app/args.go

```diff
@@ -100,9 +101,10 @@ func AddSonobuoyConfigFlag(cfg *SonobuoyConfig, flags *pflag.FlagSet) {
 }
 
 const (
-	e2eFocusFlag    = "e2e-focus"
-	e2eSkipFlag     = "e2e-skip"
-	e2eParallelFlag = "e2e-parallel"
+	e2eFocusFlag          = "e2e-focus"
+	e2eSkipFlag           = "e2e-skip"
+	e2eParallelFlag       = "e2e-parallel"
+	e2eCustomRegistryFlag = "e2e-repo-list"
```

Comment:
Same as the comment for the docs, should this be `e2e-repo-config` or something implying a config file?

Reply:
That sounds good; `repo-config` is a good balance between saying `repo-file` which wasnt as clear to me and `repo` which definitely suggested a single repo.